cmd/gardener-extension-provider-aws/app/app.go

```diff
@@ -269,6 +269,14 @@ func NewControllerManagerCommand(ctx context.Context) *cobra.Command {
 				return fmt.Errorf("could not add ready check for webhook server to manager: %w", err)
 			}
 
+			// TODO (georgibaltiev): Remove after the release of version 1.62.0
```

Comment:
Nit: In Gardener, we usually keep such migration codes for at least 3 releases.

From https://github.com/gardener/gardener/blob/v1.118.0/docs/development/process.md#deprecations-and-backwards-compatibility:
> In case you have to run migration code (_which is mostly internal_), please add a [TODO statement](#todo-statements) to remove it only after **3 releases**.

```suggestion
			// TODO (georgibaltiev): Remove after the release of version 1.64.0
```